src/Certify.Core/Certify.Core.csproj

```diff
@@ -82,6 +82,10 @@
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net8.0|AnyCPU'">
     <NoWarn>1701;1702;CA1068</NoWarn>
   </PropertyGroup>
+  <PropertyGroup>

```

Comment:
As part of our build be define a Build.props file in  certify/Directory.Build.props. Each project should pick this up for a set of common build settings. Can you check if this conditional can be moved to that and that way all these project files don't need to define the condition individually.